docs/Node Operators/configuration-variables.md

```diff
@@ -408,11 +408,13 @@ By default, Chainlink nodes write log data to `$ROOT/log.jsonl`. The log directo
 The `LOG_LEVEL` environment variable determines both what is printed on the screen and what is written to the log file.
 
 The available options are:
+- `"trace"`: Used only in development, off by default and not accessible on production builds.
 - `"debug"`: Useful for forensic debugging of issues.
 - `"info"`: High level informational messages.
-- `"warn"`: Something unexpected happened that might need non-urgent action. Node operators should check these once in a while to see whether anything stands out (e.g. deprecation warnings).
-- `"error"`: Something bad happened. Node operators might need to take urgent action based on this error.
-- `"panic"`: Something very bad happened. Node operators should take immediate action to fix this.
+- `"warn"`: A mild error occurred that might require non-urgent action. Check these warnings semi-regularly to see if any of them require attention. These warnings usually happen due to factors outside of the control of the node operator. Examples: Unexpected responded from a remote API or inaccurate networking errors.
+- `"error"`: An error occurred that might have put the node in a partially broken state. Node operators might need to take action to remedy this error. Examples: Deprecation warnings or incorrectly configured settings that cause a job to fail.
+- `"crit"`: A critical error occurred. The node might be completely stopped and non-responsive. Node operators should take immediate action to fix these errors. Examples: The node was not able to boot because a network socket could not be opened or the database became inaccessible.
+- `"panic"`: An unrecoverable error occurred that was unexpected. Node operators should try to restart their nodes, and notify the Chainlink team of a bug.
```

Comment:
> Node operators should try to restart their nodes

Why? Panics are not necessarily fatal. A task can panic and be recovered and marked as an error, without affecting any other systems.

Reply:
I imagine that there are concerns about the panic having left some part of the system in an invalid state

Reply:
If that is truly a valid concern, then we should not be recovering and stopping the system from shutting down. It doesn't make sense to recover and keep the node alive while also expecting node ops to notice and manually restart.

Reply:
Do we have a consensus on what the correct node operator response should be for `panic`? I changed it a little in my last commit:

```
If the node is unresponsive, node operators should try to restart their nodes and.....
```

Reply:
Yeah that sounds good.
I'm worried about 'unrecoverable' being too suggestive - What about:
> An exceptional error occurred that could not be handled.

Reply:
Updated. Looks like this PR is good to go if there are no other concerns.